### PR TITLE
Update sample-reads-randomly.py

### PR DESCRIPTION
**Pull request by brooksph**
Added description for -N

- [ ] Is it mergeable?
- [ ] `make test` Did it pass the tests?
- [ ] `make clean diff-cover` If it introduces new functionality in
  `scripts/` is it tested?
- [ ] `make format diff_pylint_report cppcheck doc pydocstyle` Is it well
  formatted?
- [ ] Did it change the command-line interface? Only backwards-compatible
  additions are allowed without a major version increment. Changing file
  formats also requires a major version number increment.
- [ ] For substantial changes or changes to the command-line interface, is it
  documented in `CHANGELOG.md`? See [keepachangelog](http://keepachangelog.com/)
  for more details.
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Do the changes respect streaming IO? (Are they
  tested for streaming IO?)

### CODE REVIEW

File: scripts/sample-reads-randomly.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def get_parser():
 
     parser.add_argument('filenames', nargs='+')
     parser.add_argument('-N', '--num_reads', type=int, dest='num_reads',
-                        default=DEFAULT_NUM_READS)
+                        default=DEFAULT_NUM_READS, help='samples the number of sequences or pairs specified with -N')
```
Comment (standage) [inline scripts/sample-reads-randomly.py]: khmer enforces the PEP8 style guide for all Python code. This includes a limit of 79 characters per line. You can wrap strings across multiple lines as shown below.

```python
function(arg1, arg2, arg3='if the string is really long and has a lot of words'
         ' you can wrap it across multiple lines like this')
```

You can also check for any code style violations with `make pep8` before committing.

**Comment by brooksph** [comment]
Fixed!

File: scripts/sample-reads-randomly.py [hunk 2]
```diff
@@ -86,7 +86,8 @@ def get_parser():
 
     parser.add_argument('filenames', nargs='+')
     parser.add_argument('-N', '--num_reads', type=int, dest='num_reads',
-                        default=DEFAULT_NUM_READS)
+                        default=DEFAULT_NUM_READS, help='samples the number of sequences' 
+                        'or pairs specified with -N')
```
Comment (standage) [inline scripts/sample-reads-randomly.py]: If you concatenate the two halves of the string, there's no space between "of sequences" and "or pairs".

Sorry to be so nitpicky. I can only do so after having made that mistake many...many...many times myself: :-)

**Comment by standage** [comment]
Looks good! Will merge once the CI build completes.

**Comment by brooksph** [comment]
Thank ya kindly

File: scripts/sample-reads-randomly.py [hunk 3]
```diff
@@ -86,7 +86,8 @@ def get_parser():
 
     parser.add_argument('filenames', nargs='+')
     parser.add_argument('-N', '--num_reads', type=int, dest='num_reads',
-                        default=DEFAULT_NUM_READS)
+                        default=DEFAULT_NUM_READS, help='samples the number of sequences '
```
Comment (standage) [inline scripts/sample-reads-randomly.py]: This line is still 91 characters long, and it has an empty space at the end. Things that pep8 complains about. ¯\_(ツ)_/¯

**Comment by brooksph** [comment]
Fail

**Comment by standage** [comment]
Woot

**Pull request merged**